changelog/T/TaxTaxation.md

```diff
@@ -0,0 +1,14 @@
+# What Has Changed In the “TaxTaxation” Article
+
+## Major changes
+Some words or sentences were changed because they seemed too complicated to the general understanding of the target audience.
+
+The word 'booty' was replaced by the word 'gain':
+ 
+ In this more stable period, income came from tribute but not from booty. --> In this more stable period, income came from tribute but not from gain.
+ The provinces were considered booty by the Romans and were plundered physically by the army and financially by the tax collectors.
+  --> The provinces were considered gain by the Romans and were stolen physically by the army and financially by the tax collectors.
+
+The word 'plunder' was replaced by the word 'steal':
+  The provinces were considered booty by the Romans and were plundered physically by the army and financially by the tax collectors.
+  --> The provinces were considered gain by the Romans and were stolen physically by the army and financially by the tax collectors.
```

Comment:
suggestion: "The provinces were considered as gains to take by the Romans" or "The provinces were considered as full of teasures to take by the Romans". Or similar.